code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -475,11 +475,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	color = "#3E1B00" // rgb: 62, 27, 0
 	boozepwr = 70
 	quality = DRINK_NICE
-	taste_description = "cola"
+	taste_description = "well done job"
 	glass_icon_state = "whiskeycolaglass"
-	glass_name = "whiskey cola"
+	glass_name = "Whiskey Cola"
 	glass_desc = "An innocent-looking mixture of cola and Whiskey. Delicious."
 
+/datum/reagent/consumable/ethanol/vodka_cola
+	name = "Vodka Cola"
+	description = "Vodka, mixed with cola"
+	color = "#3f2410" // rgb: 62, 27, 0
+	boozepwr = 60
+	quality = DRINK_NICE
+	taste_description = "depression"
```

Comment:
This would make a lot more sense as something like 'vodka and cola', since it isn't exactly a special drink it shouldn't really have an abstract taste_description in my opinion

Reply:
Understood, then I will redo it and add it later with other fix/features, do you see my comment if I close the pull request before you saw it ?

Reply:
Yea, there could be a few ways of getting around that if you can think of them, otherwiseit's not a major issue

Reply:
Sure, thanks